WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -397,6 +399,24 @@ private extension DefaultProductFormTableViewModel {
         let title = Constants.noPriceWarningTitle
         return ProductFormSection.SettingsRow.WarningViewModel(icon: icon, title: title)
     }
+
+    // MARK: Product downloads only
+
+    func downloadsRow(product: ProductFormDataModel) -> ProductFormSection.SettingsRow.ViewModel {
+        let icon = UIImage.cloudImage
+        let title = Constants.downloadsTitle
+        var details = Constants.emptyDownloads
+        if product.downloads.count == 1 {
+            details = String.localizedStringWithFormat(Constants.singularDownloadsFormat, product.downloads.count)
+        }
+        else if product.downloads.count > 1 {
+            details = String.localizedStringWithFormat(Constants.pluralDownloadsFormat, product.downloads.count)
+        }
```

Comment:
What do you think about:
```suggestion
        switch product.downloads.count {
        case 1:
            details = String.localizedStringWithFormat(Constants.singularDownloadsFormat, product.downloads.count)
        case 2...:
            details = String.localizedStringWithFormat(Constants.pluralDownloadsFormat, product.downloads.count)
        default:
            details = ""
        }
```

Reply:
Perfect. Changed according to your suggestion.